### ISSUE TITLE
cmake trunk sets CMAKE_INSTALL_PREFIX:PATH=/usr/local which cannot be written to (Trac #1524)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1516]
In trying `make tarball` using trunk of simulation meta-project it fails with the following error:

CMake Error at cmake_install.cmake:36 (FILE):
file cannot create directory: /usr/local/lib/icecube.  Maybe need
administrative privileges.

In looking in the CMakeCache.txt file the CMAKE_INSTALL_PREFIX:PATH is incorrectly set to /usr/local 

Migrated from https://code.icecube.wisc.edu/ticket/1524
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:12:46", 
    "description": "In trying `make tarball` using trunk of simulation meta-project it fails with the following error:\n\nCMake Error at cmake_install.cmake:36 (FILE):\nfile cannot create directory: /usr/local/lib/icecube.  Maybe need\nadministrative privileges.\n\nIn looking in the CMakeCache.txt file the CMAKE_INSTALL_PREFIX:PATH is incorrectly set to /usr/local ", 
    "reporter": "melanie.day", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251966149934", 
    "component": "cmake", 
    "summary": "cmake trunk sets CMAKE_INSTALL_PREFIX:PATH=/usr/local which cannot be written to", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-01-22T17:22:22", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160216T02:01:32: 
* nega changed status from "new" to "closed"
* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160321T20:33:00: 
* melanie.day changed status from "closed" to "reopened"
* melanie.day changed resolution from "fixed" to ""
* melanie.day commented:

> I downloaded trunk of cmake today and it is still doing this

**Comment by icecube-trac** [comment]
Trac update at 20160321T22:37:37: 
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 2487:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2487"
> force CMAKE_INSTALL_PREFIX into the cache
> 
> as a default cmake variable, CMAKE_INSTALL_PREFIX is cached by default. to
> `set()` it needs to be forced into the cache with `set(VAR VAL CACHE "docstring" FORCE)`
> or scope-wise overwrite it with `set(VAR VAL)`.
> 
> closes #1516
> ```


* nega changed status from "reopened" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:12:46: nwhitehorn commented:

> In changeset 170223:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170223"
> force CMAKE_INSTALL_PREFIX into the cache
> 
> as a default cmake variable, CMAKE_INSTALL_PREFIX is cached by default. to
> `set()` it needs to be forced into the cache with `set(VAR VAL CACHE "docstring" FORCE)`
> or scope-wise overwrite it with `set(VAR VAL)`.
> 
> closes #1516
> ```
